pkgs/top-level/all-packages.nix

```diff
@@ -6922,13 +6923,21 @@ with pkgs;
     python = python27.override{x11Support=true;};
     db = db.override { dbmSupport = true; };
   };
+  pypy3 = callPackage ../development/interpreters/python/pypy/3 {
```

Comment:
Let's make this `pypy35` and the other one `pypy27`.

Reply:
Alright, but note that upstream names its directories `2.7` and `3`, and has no intention to support 3.5 and 3.6 simultaneously.

Reply:
Interesting, I was not aware of that. You can ignore it then.